example/devstone/main_flat_devstone.cpp

```diff
@@ -0,0 +1,60 @@
+#include <chrono>
```

Comment:
What do you think about setting the flattening as an optional configuration parameter? So there is only one example and users can use the command line to flatten or not

Reply:
I agree, but we need to use macros for that.